src/test/ts/alien/Loader.ts

```diff
@@ -3,6 +3,7 @@ import { Fun } from '@ephox/katamari';
 import { Attr, Body, Element, Insert, Remove, SelectorFind } from '@ephox/sugar';
 import Editor from 'src/main/ts/index';
 
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
wild

Reply:
It seems that the import does not have types defined for it. This leads typescript to complain about an implicit 'any' type. That error was disabled by the @ts-ignore however that then triggers the linting to error so we need the line to disable that lint.